core/su_mnlp.c

```diff
@@ -19,6 +19,8 @@ uint32_t qb_50_time;
 uint32_t obc_day_moment_secs = 0;
 uint32_t tt_day_moment_secs = 0; /*script time-table moment in seconds from start of current day*/
 uint32_t moment_diff = 0;        /**/
+uint8_t sys_view_temp[40];
```

Comment:
I think its better to use the SYSVIEW_PRINT found in the core/sysview.h instead of the segger print.